extensions/dacpac/src/telemetry.ts

```diff
@@ -14,5 +14,7 @@ let packageInfo = Utils.getPackageInfo(packageJson);
 export const TelemetryReporter = new AdsTelemetryReporter(packageInfo.name, packageInfo.version, packageInfo.aiKey);
 
 export enum TelemetryViews {
-	SelectOperationPage = 'SelectOperationPage'
+	DataTierApplicationWizard = 'DataTierApplicationWizard',
```

Comment:
is the telemetry view based on the UI page where an event happens or where the code is? Because everything in this extension is happening in the DataTierApplicationWizard

Reply:
These views are based on the file, but, yeah I will update with the UI page view.